tests/ui/test_pv_encryption_ui.py

```diff
@@ -0,0 +1,292 @@
+import logging
+import os
+import time
+
+import pytest
+
+from ocs_ci.helpers.helpers import (
+    create_unique_resource_name,
+    create_pods,
+)
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    ResourceNotFoundError,
+    KMSResourceCleaneupError,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pvc import get_all_pvc_objs
+
+from ocs_ci.ocs.ui.storageclass import StorageClassUI
+
+from ocs_ci.ocs.ui.pvc_ui import PvcUI
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility import kms
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier1,
+    skipif_ocs_version,
+)
+from ocs_ci.utility.utils import get_vault_cli, get_ocp_version
+from ocs_ci.ocs import constants
+from ocs_ci.utility import version
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames=["kv_version"],
+    argvalues=[
+        pytest.param("v1", marks=pytest.mark.polarion_id("OCS-2585")),
+        pytest.param("v2", marks=pytest.mark.polarion_id("OCS-2592")),
+    ],
```

Comment:
it seems like these are already exsisting polarion IDs - TestRbdPvEncryption

Reply:
@ebenahar That's true, so shouldn't we use them?

Reply:
Since these are new tests we should have new Polarion IDs

Reply:
@ebenahar Done, pls approve once more.